src/Commands/Servers/Make.php

```diff
@@ -39,9 +42,12 @@ public function configure()
             ->addOption('private-ip', null, InputOption::VALUE_REQUIRED, 'Private IP address of the server. Required when using the "custom" provider.')
             ->addOption('php', null, InputOption::VALUE_REQUIRED, 'PHP version to install when provisioning the server. Supported versions are "php56", "php70", and "php71".')
             ->addOption('database', null, InputOption::VALUE_REQUIRED, 'The name of the database to create when provisioning the server.', 'forge')
+            ->addOption('database-type', null, InputOption::VALUE_OPTIONAL, 'The type of DB to install. Can be "mariadb", "mysql", "mysql8, "postgres", or "none".', 'mysql')
```

Comment:
```suggestion
            ->addOption('database-type', null, InputOption::VALUE_OPTIONAL, 'The type of database to install. Can be "mariadb", "mysql", "mysql8, "postgres", or "none".', 'mysql')
```

Instead of allowing `"none"` here and adding the `if` down in the `execute` method, we can also default to `''`. What do you think?

Reply:
I'm not actually sure this even works. I've tried it via the API and I cannot seem to provision a server without a DB. Perhaps I am missing something here.